pysat/_constellation.py

```diff
@@ -49,8 +59,13 @@ class Constellation(object):
 
     Attributes
     ----------
-    instruments : list
-        A list of pysat Instruments that make up the Constellation
+    platforms
+    names
+    tags
+    inst_ids
+    instruments
+    index_res
+    common_index
```

Comment:
Followed the style guidlines here: https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes

I don't believe we've done this in the other classes, but it might be worth doing.

Reply:
I agree. We do list Attributes in at least some of the other classes. Those don't make use of the simplification, as you've noted.